cmd/storage.go

```diff
@@ -37,20 +40,37 @@ var storageCreateCmd = &cobra.Command{
 	Example: `  # Create storage of size 1Gb to a component
   odo storage create mystorage --path=/opt/app-root/src/storage/ --size=1Gi
 	`,
-	Args: cobra.ExactArgs(1),
+	Args: cobra.MaximumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		client := getOcClient()
 		applicationName, err := application.GetCurrent(client)
 		checkError(err, "")
 		projectName := project.GetCurrent(client)
 		componentName := getComponent(client, storageComponent, applicationName, projectName)
+		currentComponentName, err := component.GetCurrent(client, applicationName, projectName)
```

Comment:
We are retrieving the component name in the last line above, so is this line required?

Reply:
yeah, not required :(